app/helpers/application_helper.rb

```diff
@@ -29,11 +29,15 @@ def event_status_icon(event)
   end
 
   def event_progress_color(progress)
-    case progress.to_i
-    when 100 then 'progress-bar-success'
-    when 85 then 'progress-bar-info'
-    when 71 then 'progress-bar-warning'
-    when 14, 28, 42, 57 then 'progress-bar-danger'
+    progress = progress.to_i
```

Comment:
You could rewrite case like this http://stackoverflow.com/a/6208739
@ChrisBr which way would you prefer?